packages/@dcl/inspector/src/components/EntityInspector/TriggerInspector/TriggerCondition/TriggerCondition.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { AiOutlinePlus as AddIcon } from 'react-icons/ai'
+import { VscTrash as RemoveIcon } from 'react-icons/vsc'
+import { TriggerConditionType } from '@dcl/asset-packs'
+
+import Button from '../../../Button'
+import { Dropdown } from '../../../Dropdown'
+import MoreOptionsMenu from '../../MoreOptionsMenu'
+
+import type { Props } from './types'
+
+import './TriggerCondition.css'
+
+export const TriggerCondition = ({
+  trigger,
+  availableStates,
+  onChangeEntity,
+  onChangeConditionType,
+  onChangeConditionValue,
+  onAddTriggerCondition,
+  onRemoveTriggerCondition
+}: Props) => {
+  const conditionTypeOptions = [
+    { value: TriggerConditionType.WHEN_STATE_IS, text: 'is' },
+    { value: TriggerConditionType.WHEN_STATE_IS_NOT, text: 'is not' }
+  ]
+  return (
+    <div className="TriggerConditionsContainer">
+      <div className="TriggerConditionsTitle">
+        <span>Trigger Condition</span>
+        <div className="RightContent">
+          <Button className="AddButton" onClick={onAddTriggerCondition}>
+            <AddIcon size={16} />
+          </Button>
+        </div>
+      </div>
+      {trigger.conditions!.map((condition, idx) => {
```

Comment:
I would make sure that the `conditions` here is defined, because the type does not enforce it. It could happen for example that a smart item defined in the catalog has a trigger without the property `conditions` defined and it would break here.

Reply:
good catch! fixed it